src/js/config/config.js

```diff
@@ -14,7 +14,7 @@ var config = {
   }
 };
 if (config.environment === "production") {
-  config.rootUrl = "dist/";
+  config.rootUrl = "assets/";
 }
 if (process.env.GULP_ENV === "development") {
```

Comment:
We can't! Here's why:

- config.environment is checked at "runtime" (for example by Marathon.jsx to build the logo Path)
- process.env.GULP_ENV is checked during "build time" 

I will add a comment. Also, we can get rid of `config.rootUrl` since now we always serve from within the same folder also in production.